Main.qml

```diff
@@ -7,19 +7,34 @@ import QtQuick.Controls.Universal
 
 Window {
     width: 640
-    height: 480
+    height: 750
     visible: true
     title: qsTr("LivePose_test")
 
     DataModel {
         id: x
         Component.onCompleted:
         {
+            // Set pose_backend as backend
+            x.pose_backend.name = "PoseNet";
+            x.pose_backend.params = x.posenet_params;
+            x.config.pose_backends = [ x.pose_backend ];
+
+            // Set cameras
+            x.cameras.camera1 = "Camera 1";
+            x.cameras.camera2 = "Camera 2";
+            x.config.cameras = x.cameras;
+
+            // Set outputs
```

Comment:
I think for the output you also want a list of variants in your Config as a given config file does not necessarily have one instance of every single output type (or can have multiple of the same type: for instance two OSC outputs sending to different IP addresses)